### PR TITLE
Write to feeder node, and address review comments

### PR DESCRIPTION
**Pull request by jonmv**
@geirst please review.

### CODE REVIEW

**Comment by jonmv** [comment]
Ah, ruby devz ...

File: tests/performance/reindexing/reindexing_and_feeding.rb [hunk 1]
```diff
@@ -5,150 +5,112 @@
...
+		gateway(ContainerDocumentApi.new)).
+    admin_metrics(Metrics.new).
+    indexing("combinedcontainer").
+    sd(selfdir + "doc.sd")
+
     deploy_app(@app)
     start
 
+    @qrserver = @vespa.container["combinedcontainer/0"]
+    @document_count = 1 << 19
```
Comment (geirst) [inline tests/performance/reindexing/reindexing_and_feeding.rb:41]: I prefer an explicit number here.

File: tests/performance/reindexing/reindexing_and_feeding.rb [hunk 2]
```diff
@@ -193,21 +155,75 @@ def get_reindexing_status
...
   end
 
+  def graph_config
+    {
+      'reindex' => {
+	'reindexing.throughput'   => { :y_min =>  10000, :y_max =>  50000 },
+	'reindexing.time.seconds' => { :y_min =>     30, :y_max =>    120 }
+      },
+      'feed' => {
+	'qps' =>                     { :y_min =>  10000, :y_max =>  50000 },
```
Comment (geirst) [inline tests/performance/reindexing/reindexing_and_feeding.rb:165]: Is it feed throughput you want here? If so, _feeder.throughput_ is the key used when generating the report.

Comment (jonmv) [reply tests/performance/reindexing/reindexing_and_feeding.rb:165]: Thanks, yes. `qps` is used by fbench :)

File: tests/performance/reindexing/reindexing_and_feeding.rb (hunk 1 above)
Comment (jonmv) [reply tests/performance/reindexing/reindexing_and_feeding.rb]: I like powers of two, and so does bucket distribution, but OK :)

**Pull request merged**